components/toast/stories/toast.test.js

```diff
@@ -24,6 +24,18 @@ export const ToastGroup = Variants({
 			variant: "positive",
 			message: "Copied to clipboard",
 			inlineButtonLabel: "Eject",
-		}
+		},
+		{
+			testHeading: "Wrapping with button",
+			variant: "info",
+			message: "A new version of Lightroom Classic is now available. Use the Update button below to start using the new version.",
+			inlineButtonLabel: "Update"
+		},
+		{
+			testHeading: "Wrapping without button",
+			variant: "info",
+			message: "A new version of Lightroom Classic is now available. Use the Update button below to start using the new version.",
+			inlineButtonLabel: ""
+		}	
```

Comment:
I love that you added this "Wrapping without button" - maybe we could add one more test case without the button with a shorter, non-wrapping label?